src/renderer/actions/register.ts

```diff
@@ -10,6 +10,23 @@ import {KEYSTORE_DEFAULT_DIRECTORY} from "../constants/keystore";
 import {CGAccount} from "../models/account";
 import {getV4Filename} from "../services/utils/crypto-utils";
 
+// Mnemonic Failed Verification
+export const storeFailedVerificationAction = (failedVerification: boolean) =>
+    (dispatch: Dispatch<IFailedVerificationAction>): void => {
+        dispatch(setFailedVerification(failedVerification));
+    };
+
+export const setFailedVerification = (failedVerification: boolean): IFailedVerificationAction => ({
```

Comment:
storeSigningMnemonicVerificationStatus

- rename on other places as well to properly describe action